mLRS/CommonTx/mlrs-tx.cpp

```diff
@@ -811,16 +839,26 @@ INITCONTROLLER_END
         break;
 
     case LINK_STATE_TRANSMIT:
+        do_transmit_prepare(tdiversity.Antenna());
+        link_state = LINK_STATE_TRANSMIT_SEND;
+        DBG_MAIN_SLIM(dbg.puts("\nt");)
+        break;
+
+    case LINK_STATE_TRANSMIT_SEND: {
+        uint16_t dt = micros16() - pretransmit_tstamp_us;
+        if (dt < 500) break;
```

Comment:
It appears you want to send at least 500 us after tx_tick == 1.  How is this supposed to reduce jitter?  Why aren't we syncing to doSysTask here?

Reply:
we are not synching to tx anymore becasue when we did the gap had been 1 ms ... now it can be shorter, here 500 us

Reply:
OK.  So, should I test 500 or 750 which JLP says was better?

Reply:
I guess I would first go with 500
it is not yet clear to me why 500 was a regression for JLP
if you also see 500 us isn't good but 750 is, I guess it's maybe not worth the 250us and should undo the last commit